tests/components/meteo_france/test_config_flow.py

```diff
@@ -1,29 +1,56 @@
 """Tests for the Meteo-France config flow."""
-from meteofrance.client import meteofranceError
+from meteofrance.model import Place
 import pytest
 
 from homeassistant import data_entry_flow
 from homeassistant.components.meteo_france.const import CONF_CITY, DOMAIN
 from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
 
 from tests.async_mock import patch
 from tests.common import MockConfigEntry
 
 CITY_1_POSTAL = "74220"
 CITY_1_NAME = "La Clusaz"
-CITY_2_POSTAL_DISTRICT_1 = "69001"
-CITY_2_POSTAL_DISTRICT_4 = "69004"
-CITY_2_NAME = "Lyon"
-
-
-@pytest.fixture(name="client_1")
-def mock_controller_client_1():
+CITY_1_LAT = 45.90417
+CITY_1_LON = 6.42306
+CITY_1_COUNTRY = "FR"
+CITY_1_ADMIN = "Rhône-Alpes"
+CITY_1_ADMIN2 = "74"
+
+CITY_2_NAME = "Auch"
+CITY_2_LAT = 43.64528
+CITY_2_LON = 0.58861
+CITY_2_COUNTRY = "FR"
+CITY_2_ADMIN = "Midi-Pyrénées"
+CITY_2_ADMIN2 = "32"
+
+CITY_3_NAME = "Auchel"
+CITY_3_LAT = 50.50833
+CITY_3_LON = 2.47361
+CITY_3_COUNTRY = "FR"
+CITY_3_ADMIN = "Nord-Pas-de-Calais"
+CITY_3_ADMIN2 = "62"
```

Comment:
Save `Place`s directly here, so we are not dependant on th lib format of a place (will avoid `f"{CITY_3_NAME} - {CITY_3_ADMIN} ({CITY_3_ADMIN2}) - {CITY_3_COUNTRY}"`, will be `f"{CITY_1}"`)

Use the Place instance to check the longitude and latitude

Reply:
done